src/core/blockchain/zilliqa/client.ts

```diff
@@ -103,6 +107,26 @@ export class Client extends BlockchainGenericClient {
         return feeOptions;
     }
 
+    public async getSmartContractSubState(
+        contractAddress: string,
+        field: string,
+        subFields: string[] = []
+    ) {
+        return this.call('GetSmartContractSubState', [
+            fromBech32Address(contractAddress)
+                .replace('0x', '')
+                .toLowerCase(),
+            field,
+            subFields
+        ]).then(response => response?.result);
+    }
+
+    public async getSmartContractInit(address: string) {
```

Comment:
this method could be used with both formats, i would do this:
- check if address is a bech32 address
  - if yes call fromBech32
- remove 0x and call toLowerCase()